dwave/system/samplers/leap_hybrid_sampler.py

```diff
@@ -304,22 +357,82 @@ def parameters(self):
             return parameters
 
     def sample_dqm(self, dqm, time_limit=None, compressed=False, **kwargs):
-        """Sample from the specified binary quadratic model.
+        """Sample from the specified discrete quadratic model.
 
         Args:
-            bqm (:obj:`dimod.DiscreteQuadraticModel`):
-                The binary quadratic model.
+            dqm (:obj:`dimod.DiscreteQuadraticModel`):
+                Discrete quadratic model (DQM).
+
+            time_limit (int, optional):
+                Maximum run time, in seconds, to allow the solver to work on the
```

Comment:
Again, way too much information. At the very least, this should refer to the method and document the behaviour there.